models/property.js

```diff
@@ -0,0 +1,30 @@
+const mongoose = require('mongoose');
+const Property =  mongoose.model('Property', new mongoose.Schema({
+    name: {
+        type: String,
+        required: true,
+        minlength: 3,
+        maxlength: 250
+
+    },
+    description:{
+        type: String,
+        required: true,
+        minlength: (5),
```

Comment:
Any reason for putting the values of `minlength` and `maxlength` in braces?

Reply:
No sir .